python/Ganga/new_tests/GPI/GangaList/TestCopy.py

```diff
@@ -43,7 +36,7 @@ def testCopy(self):
         gl2 = copy.copy(gl)
         assert len(gl2) == len(gl), 'lists must be equal'
         assert gl2 is not gl, 'list must be copies'
-        #assert gl[0] is gl2[0], 'the references must be copied' ## rcurrie 09/12/15 Not sure this is valid given we have to cope with GangaObjects here which may not be copyable!
+        assert gl[0] is gl2[0], 'the references must be copied'
```

Comment:
so what is the parent of gl[0] and what is the parent of gl2[0]?
I think we expect in this case that the parent to both be None.
How about in the case of the GangaList being a member of a class object:
i.e.:
```
j=Job(); j.someArg = GangaList()
gl2 = copy.copy(j.someArg)
assert stripProxy(j.someArg[0])._getParent() is stripProxy(j)
assert stripProxy(j.someArg[0])._getParent() is not stripProxy(gl2[0])._getParent()
assert stripProxy(gl2[0])._getParent() is None
```